python/SWESimulators/Simulator.py

```diff
@@ -89,6 +167,38 @@ def downloadPrevTimestep(self):
         """
         return self.cl_data.downloadPrevTimestep(self.cl_queue)
         
+    def copyState(self, otherSim):
+        """
+        Copies the state ocean state (eta, hu, hv) from the other simulator.
+        
+        This function is exposed to enable efficient re-initialization of
+        resampled ocean states. This means that all parameters which can be 
+        initialized/assigned a perturbation should be copied here as well.
```

Comment:
Is wind part of the state? Why not bathymetry? (We might do perturbations on bathymetry at some stage.) Maybe we should write it as "state and wind"?

Reply:
You illustrate my point: "We **might** do perturbations on bathymetry". We might also perturb other things! So my suggestion is that when we perturb anything new, we should never expect that this parameter is handled specially before. As a consequence: Bathymetry should therefore be added in this function when we actually perturb it.